### DISCUSSION TITLE
Mettre à jour une carte à partir d'un fichier en ligne

### DISCUSSION THREAD

**Discussion by opetitprestataire** [IGNF-Ma-carte/.github#10] [Général]
Bonjour, 
Je dispose d'un fichier avec les adresses des implantations de mon organisme qui est maintenu dans un google doc.
J'ai testé la technique du geocodage qui fonctionne bien avec un fichier.
Existe-t-il un moyen de conserver le lien avec le google doc d'origine pour que les modifications du google doc se répercutent directement sur la carte ?
Merci beaucoup

**Comment by Viglino** [comment]
Non, désolé.
Vous devez refaire le géocodage si le fichier change...
